server/api/controllers/employers.py

```
@@ -0,0 +1,103 @@
+            "user_id": user.id,
+            "full_name": user.full_name,
+            "email": user.email,
+            "phone": user.phone,
+            "location": user.location,
+            "bio": user.bio,
+            "company_name": user.company_name,
+            "website": user.website,
+            "image": user.image,
+        }), 200
+        
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)